stable_baselines3/common/recurrent/policies.py

```diff
@@ -615,3 +621,191 @@ def __init__(
             enable_critic_lstm,
             lstm_kwargs,
         )
+
+
+class RecurrentFeaturesExtractorActorCriticPolicy(BaseRecurrentActorCriticPolicy, Generic[ExtractorInput, RecurrentState]):
+    features_extractor: RecurrentFeaturesExtractor[ExtractorInput, RecurrentState]
+
+    def __init__(
+        self,
+        observation_space: spaces.Space,
+        action_space: spaces.Space,
+        lr_schedule: Schedule,
+        net_arch: Optional[Union[List[int], Dict[str, List[int]]]] = None,
+        activation_fn: Type[nn.Module] = nn.Tanh,
+        ortho_init: bool = True,
+        use_sde: bool = False,
+        log_std_init: float = 0.0,
+        full_std: bool = True,
+        use_expln: bool = False,
+        squash_output: bool = False,
+        features_extractor_class: Type[BaseFeaturesExtractor] = GRUNatureCNNExtractor,
+        features_extractor_kwargs: Optional[Dict[str, Any]] = None,
+        share_features_extractor: bool = True,
+        normalize_images: bool = True,
+        optimizer_class: Type[th.optim.Optimizer] = th.optim.Adam,
+        optimizer_kwargs: Optional[Dict[str, Any]] = None,
+    ):
+        if features_extractor_kwargs is None:
+            features_extractor_kwargs = {}
+        # Automatically deactivate dtype and bounds checks
+        if normalize_images is False and issubclass(features_extractor_class, GRUNatureCNNExtractor):
+            features_extractor_kwargs = features_extractor_kwargs.copy()
+            features_extractor_kwargs.update(dict(normalized_image=True))
+
+        if not issubclass(features_extractor_class, RecurrentFeaturesExtractor):
+            base_features_extractor = features_extractor_class(observation_space, **features_extractor_kwargs)
+
+            features_extractor_class = GRUWrappedFeaturesExtractor
+            new_features_extractor_kwargs = dict(base_extractor=base_features_extractor)
+            if "features_dim" in features_extractor_kwargs:
+                new_features_extractor_kwargs["features_dim"] = features_extractor_kwargs["features_dim"]
+            features_extractor_kwargs = new_features_extractor_kwargs
+            print(features_extractor_class, features_extractor_kwargs)
+
+        super().__init__(
```

Comment:
Ultimately up to you, but you may want to consider setting all of these as kwargs instead of args. With args, its easy to flip two parameters whose types match and thereby introduce weird configuration bugs. I feel like this happens more often with long param lists like this.

Ie:
```
super().__init__(
            observation_space=observation_space,
            action_space=action_space,
            ...
```

Reply:
Thanks! I was being lazy because my editor autocompleted it like this. Fixed in 750d78e9